nsinit/exec.go

```diff
@@ -143,6 +144,14 @@ func startContainer(container *libcontainer.Config, dataPath string, args []stri
 		return cmd
 	}
 
+	setupCommand := func(container *libcontainer.Config, console, dataPath, init string) *exec.Cmd {
+		setupCmd = namespaces.DefaultSetupCommand(container, console, dataPath, init)
+		if logPath != "" {
+			cmd.Env = append(cmd.Env, fmt.Sprintf("log=%s", logPath))
```

Comment:
This all stuff is little confusing for me. Why we setup `log` for cmd in `setupCommand`. Also, I wondering why we need to do this in closure? I think we can just setup it here and pass `setupCmd` and `cmd` directly.